gslib/gcs_json_api.py

```diff
@@ -228,7 +231,18 @@ def __init__(self,
 
     self.http_base = 'https://'
     gs_json_host = config.get('Credentials', 'gs_json_host', None)
-    self.host_base = gs_json_host or 'storage.googleapis.com'
+    if context_config.get_context_config(
```

Comment:
Nit: You can wrap the if condition within parenthesis to get rid of the weird formatting here.

Reply:
Also, given that context_config.create_context_config is always called at the beginning, do we need to check for context_config.get_context_config? Should that object always exist?

Reply:
The first check is for tests that may not run gsutil's main setup.